pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -1154,17 +1115,13 @@ func (b *Build) setupCosignVolumes(secrets []corev1.Secret) ([]corev1.Volume, []
 }
 
 var (
-	lowestSupportedPlatformVersion = semver.MustParse("0.3")
+	lowestSupportedPlatformVersion = semver.MustParse("0.7")
```

Comment:
I wonder if this var should've really been named `platformAPIEquals03`, because it's only used to determine if the `--process-type` and `-report` flags are passed to the exporter. 

@joe-kimmel-vmw do you know if these flags are guaranteed to be supported >0.7? If so we can also clean up line 553 thru 569 and remove this var.

 https://github.com/buildpacks-community/kpack/blob/37a9406669b68f5285895501d4afbce0a3b27a7d/pkg/apis/build/v1alpha2/build_pod.go#L553-L569

Reply:
cool,  good point.  yeah - maybe they're not guaranteed to be supported until the heat death of the universe, but for the foreseeable future they should be in there afaik.

Reply:
btw this should be fine but i did have to change the unit tests just a little more.   `proces-type=web` is the default, so now we're just passing in the default explicitly rather than sometimes silently letting the default happen.  should be NBD but it's always possible there's some tripping hazard I'm not seeing :)